src/containers/SideNav/SideNav.js

```diff
@@ -155,6 +155,13 @@ export class SideNav extends Component {
           <SideNavMenuItem element={NavLink} icon={<span />} to="/secrets">
             Secrets
           </SideNavMenuItem>
+          <SideNavMenuItem
+            element={NavLink}
+            icon={<span />}
+            to="/triggerTemplates"
+          >
+            Trigger Templates
```

Comment:
TriggerTemplates (no space, consistent with other links in the nav and throughout the dashboard, uses the plural of the kubernetes resource kind)